Dockerfile

```diff
@@ -22,13 +22,15 @@ WORKDIR /usr/src/app
 #============
 FROM base as tsbuild
 
-# use buildin python to faster build
-RUN install-apt build-essential python3
```

Comment:
Why not python here now?

Reply:
nope, `re2` now has prebuild binaries 🙃 So removed to speedup